lineman/app/components/group_page/group_page.haml

```diff
@@ -27,6 +27,7 @@
         .col-sm-8
           %section.group-page__description{aria-labelledby: 'description-card-title'}
             %h2#description-card-title.lmo-card-heading{ translate: 'group_page.description'}
+            .group-page__description-placeholder.group-page-placeholder{translate: 'group_page.description_placeholder', ng-if: 'groupPage.showDescriptionPlaceholder()' }
```

Comment:
no biggie but I was thinking the utility class for this should be .lmo-hint or something? I expect it will be repeated throughout the app